src/Widgets/FormHiddenFieldWidget/FormHiddenFieldWidgetEditingConfig.js

```diff
@@ -17,7 +17,26 @@ Scrivito.provideEditingConfig("FormHiddenFieldWidget", {
   initialContent: {
     customFieldName: "custom_hidden_field",
   },
-  validations: [customFieldNameValidation],
+  validations: [
+    customFieldNameValidation,
+    (widget) => {
+      if (
+        widget
+          .container()
+          .get("hiddenFields")
+          .map((w) => w.id())
+          .includes(widget.id())
+      ) {
+        return;
+      }
+
+      return {
+        message:
+          "Usually hidden fields should be added in the widget properties of the form container.",
```

Comment:
* [ ] "Usually" implies that there are exceptions. I think this is more confusing that helpful. Also, "widget" may be left out since it's always a widget. In the UI we call it "Form", not "Form Container". How about "Hidden fields should be added in the properties of the form.". Also not sure about in/to/at here.